railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -30,10 +30,11 @@ def edit
         say "New credentials encrypted and saved."
       end
 
-      def show
+      def show(path = nil, *)
```

Comment:
Why the splat? Particularly if we support a specific positional parameter, it seems important to complain about unexpected extras.